### ISSUE TITLE
Selection word count

### ISSUE THREAD

**Issue by ElDuderino321** [Zettlr/Zettlr#5369]
### Checklist

- [X] Zettlr does not possess this feature (if it does, but the feature doesn't work out for you, propose an enhancement instead.)
- [X] Zettlr's current features cannot be used to approximate this behaviour (otherwise, propose an enhancement instead.)
- [X] I have searched the documentation to make sure I did not overlook it.
- [X] I have searched both open and closed issues to see if someone has already proposed this.

### Feature Proposal

In Word, Libreoffice, and many other editors, you can select a portion of the text to see the word count. In Zettlr, you can only see the overall word count.

### Example Workflow

In my writing process, like many, I often start with notes and bullet points, and eventually begin writing paragraphs. Since I am often aiming for a specific word count, it would be nice to be able to see how many words my fully formulated paragraphs contain.

### Caveats

I don't see any caveats!

### Do you wish to attempt implementing this yourself?

No

### Platforms

- [X] Windows
- [X] macOS
- [X] Linux

**Comment by nathanlesage** [comment]
That shouldn't be the case. Let's reopen that as a potential bug.

**Comment by benoitvalery** [comment]
Hi, same thing here. With either word or letter count, the toolbar *always* displays the total. Zettlr 3.4.2, Debian GNU/Linux 11 (bullseye). Didn’t think it should behave differently.

**Comment by sorenss** [comment]
No, the top toolbar shows the selected *character* count for me, not the word count.

EDIT: Wait, I'm so sorry, I did not realize I had "show character count" on. I'm probably a bit confused why I can't have both at the same time (as sometimes I need one and sometimes another).

**Comment by benniekiss** [comment]
Closing as this works as expected, and comments are all related to user error or confusion
